sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detections/apriltag_grid_detection.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         # Compute the object points
         self.tags = tags
 
+        # cSpell:ignore hsize
```

Comment:
Is `hsize` acceptable in the dictionary for calibration tools, maybe?

Reply:
Done !

Reply:
so remove here?